lib/ember-test-helpers/test-module-for-acceptance.js

```diff
@@ -2,18 +2,18 @@ import AbstractTestModule from './abstract-test-module';
 import Ember from 'ember';
 import { getContext } from './test-context';
 
-export default AbstractTestModule.extend({
+export default class extends AbstractTestModule {
   setupContext() {
-    this._super({ application: this.createApplication() });
-  },
+    super({ application: this.createApplication() });
```

Comment:
This should be `super.setupContext` (bare `super` is only valid in `constructor`).